src/renderer/screens/nft/Gallery/TokensList/Item.js

```diff
@@ -94,20 +99,34 @@ const Row = ({ contract, tokenId, id, mode, account, withContextMenu = false }:
             </Text>
           </Skeleton>
           <Skeleton width={180} minHeight={24} barHeight={6} show={show}>
-            <Text ff="Inter|Medium" color="palette.text.shade50" fontSize={isGrid ? 3 : 2}>
-              <Trans
-                i18nKey="NFT.gallery.tokensList.item.tokenId"
-                values={{ tokenId: centerEllipsis(tokenId) }}
-              />
-            </Text>
+            <Box horizontal justifyContent="space-between">
+              <Text ff="Inter|Medium" color="palette.text.shade50" fontSize={isGrid ? 3 : 2}>
+                <Trans
+                  i18nKey="NFT.gallery.tokensList.item.tokenId"
+                  values={{ tokenId: centerEllipsis(tokenId) }}
+                />
+              </Text>
+              {nft.collection.standard === "ERC1155" && isGrid && (
+                <Text ff="Inter|Medium" color="palette.text.shade50" fontSize={3}>
+                  {`x${nft.amount.toString()}`}
```

Comment:
Try to prefer `toFixed` rather than `toString` when using BigNumber to avoid the scientific format with exponential in general. 🙏

Reply:
Done